instant-xml/tests/scalar.rs

```diff
@@ -71,6 +73,7 @@ fn scalars() {
             },
             cow: Cow::from("123"),
             option: Some("asd"),
+            slice: [1, 2, 3][..].into(),
```

Comment:
As we are talking about `Cows` here, might it make sense to explicitly define the cow here? Calling `into()` pretty much hides if the resulting `Cow` is owned or borrowed.

Reply:
Made these explicit.